ChangeLog

```diff
@@ -8,6 +8,8 @@ Features
      https://sweet32.info/SWEET32_CCS16.pdf.
 
 Bugfix
+   * Fix 1-byte buffer overflow in mbedtls_mpi_write_string() when
```

Comment:
I have just noticed this issue:
We usually add the entries at the end of the section, as it makes it easier for the ChangeLog merge script. However, I wouldn't block my approval for this